gradle.properties

```
@@ -16,3 +16,4 @@ kotlin.code.style=official
 android.useAndroidX=true
 android.enableJetifier=true
 android.databinding.enableV2=true
+apiKey=INSERT_YOUR_API_KEY_HERE
```

```suggestion
API_KEY=INSERT_YOUR_API_KEY_HERE
```
hằng số nên đặt tên ntn